src/css/main.css

```
@@ -0,0 +1,57 @@
+.gradient-text {
```

Automated Code Review Suggestion:

```suggestion
.chat-input { padding: 15px; border-top: 1px solid #eee; border-radius: 0 0 15px 15px; }
```
Add border-radius to the bottom corners of the chat input for visual consistency with the rest of the chat window.